build.sh

```diff
@@ -83,4 +73,4 @@ fi
 cp -R third_party/docker/mkimage* mkdebootstrap/
 
 envsubst < mkdebootstrap/Dockerfile.in > mkdebootstrap/Dockerfile
-gcloud container builds submit . --config="$CONFIG" --verbosity=info --gcs-source-staging-dir="gs://$BUCKET/staging" --gcs-log-dir="gs://$BUCKET/logs" --substitutions=_REPO="$REPO",_TAG="$TAG",_VERSION="$VERSION",_VERSION_NUMBER="$VERSION_NUMBER"
+gcloud container builds submit . --config="$CONFIG" --verbosity=info --substitutions=_REPO="$REPO",_TAG="$TAG",_VERSION="$VERSION",_VERSION_NUMBER="$VERSION_NUMBER"
```

Comment:
gcloud will pick `gs://gcp-runtimes_cloudbuild/source` for the staging dir (not the same as what you have). just making sure you're aware, if you have code that expects the original source the that will break